Core/src/main/java/com/plotsquared/core/configuration/Settings.java

```diff
@@ -249,6 +249,8 @@ public static class UUID {
         public static int UUID_CACHE_SIZE = 100000;
         @Comment("Rate limit (per 10 minutes) for background UUID fetching from the Mojang API")
         public static int BACKGROUND_LIMIT = 200;
+        @Comment("Whether the Mojang API service is enabled for inpromto (requires restart)")
```

Comment:
*impromptu

I'd also explain that these are blocking HTTP requests

Reply:
implemented in new commit